documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -358,43 +358,67 @@ as you can include your favourite version of Hibernate OGM directly to the colle
 
-Hibernate OGM {hibernate-ogm-version} requires Hibernate ORM {hibernate-orm-version} that is not included
-with WildFly 10.
+Hibernate OGM {hibernate-ogm-version} requires Hibernate ORM {hibernate-orm-version}
+and Hibernate Search {hibernate-search-verision}.
 
-For this reason, you also need the pre-packaged module ZIP for Hibernate ORM, you can download it
-https://repo1.maven.org/maven2/org/hibernate/hibernate-orm-modules/{hibernate-orm-version}/hibernate-orm-modules-{hibernate-orm-version}-{hibernate-wildfly-classifier}.zip[from Maven Central]
+These versions are not included in WildFly 10 but you can download the pre-package ZIP modules from 
+Maven Central:
 
-Unpack the archives into the `modules` folder of your WildFly 10 installation.
+* https://repo1.maven.org/maven2/org/hibernate/hibernate-orm-modules/{hibernate-orm-version}/hibernate-orm-modules-{hibernate-orm-version}-{hibernate-wildfly-classifier}.zip[Hibernate ORM modules]
+* http://repo1.maven.org/maven2/org/hibernate/hibernate-search-modules/{hibernate-search-version}/hibernate-search-modules-{hibernate-search-version}-{hibernate-wildfly-classifier}.zip[Hibernate Search modules]
+
+Once you have the archives, you need to 
+unpack them into the `modules` folder of your WildFly 10 installation.
 The modules included are:
 
 * _org.hibernate.ogm_, the core Hibernate OGM library.
 * _org.hibernate.ogm.<%DATASTORE%>_, one module for each datastore, with _<%DATASTORE%>_ being one of _infinispan_, _mongodb_ etc.
 * _org.hibernate.orm_, the Hibernate ORM libraries.
+* _org.hibernate.search_, the Hibernate Search libraries.
 * Several shared dependencies such as _org.hibernate.hql:<%VERSION%>_ (containing the query parser) and others
 
-The module slot to use for Hibernate OGM {hibernate-ogm-version} is `{hibernate-ogm-module-slot}` as the format of the slot name does not include the "micro" part of the project version.
+The module slot to use for Hibernate OGM {hibernate-ogm-version} is `{hibernate-ogm-module-slot}`
+as the format of the slot name does not include the "micro" part of the project version.
 
-To upgrade the Hibernate ORM version you will need to add the following property to your `persistence.xml`:
+You will also need to set the property `wildfly.jpa.hibernate.search.module`
+to `org.hibernate.search.orm:{hibernate-search-module-slot}` in your `persistence.xml`.
+This way your application will use the right Hibernate ORM and Hibernate Search version
+(and not the one shipped with WildFLy).
 
-.Property to set a different Hibernate ORM version
+.Property for enabling a Hibernate Search and Hibernate ORM version compatible with Hibernate OGM
 ====
 [source, XML]
 [subs="verbatim,attributes"]
 ----
-<property name="jboss.as.jpa.providerModule" value="org.hibernate:{hibernate-orm-major-minor-version}"/>
+<property name="wildfly.jpa.hibernate.search.module" value="org.hibernate.search.module:{hibernate-search-module-slot}"/>
```

Comment:
> Should the value read org.hibernate.search.orm or org.hibernate.search.module. The documentation above says one while the example uses another. Which one is right?

Damn, writing documentation is hard :/

The complete value should be `org.hibernate.search.orm:5.6.1.Final-orm51` and it's the the complete identifier of the module containing Hibernate Search ORM. You can find an explanation of the property on the WildFLy documentation: https://docs.jboss.org/author/display/WFLY10/JPA+Reference+Guide#JPAReferenceGuide-UsingHibernateSearch

Note that the -orm51 is important because it make sure that you are going to use a version compatible with ORM 5.1

Some details are also included in the Hibernate Search doc: https://docs.jboss.org/hibernate/search/5.6/reference/en-US/html_single/#_update_and_activate_latest_hibernate_search_version_in_wildfly

Sorry for the inconvinience. 

Please, let me know if you have other issues.